bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "angular-autocomplete",
   "main": "angular-autocomplete.js",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
You could keep the version at 0.0.0 until your very first actual Bower publication.